processor/integrations/integrations.go

```diff
@@ -38,113 +54,38 @@ const (
 	PostDataXML
 )
 
-//PostParameterT  has post related parameters, the URL and the data type
-type PostParameterT struct {
-	URL           string
-	Type          int
-	UserID        string
-	Payload       interface{}
-	Header        interface{}
-	RequestConfig interface{}
-}
-
-// PostParameterNewT emulates parameters needed tp make a request
-type PostParameterNewT struct {
-	Type          string
-	URL           string
-	RequestMethod string
-	UserID        string
-	Headers       interface{}
-	QueryParams   interface{}
-	Body          interface{}
-	Files         interface{}
-}
-
-// GetResponseVersion Get version of the transformer response
-func GetResponseVersion(response json.RawMessage) string {
-	parsedResponse := gjson.ParseBytes(response)
-	if parsedResponse.Get("output").Exists() {
-		return "-1"
-	}
-	if !parsedResponse.Get("version").Exists() {
-		return "0"
-	}
-	version, ok := parsedResponse.Get("version").Value().(string)
-	if !ok {
-		panic(fmt.Errorf(""))
-	}
-	return version
-}
-
-// GetPostInfoNew parses the transformer response
-func GetPostInfoNew(transformRaw json.RawMessage) PostParameterNewT {
-	var postInfo PostParameterNewT
-	var ok bool
-	parsedJSON := gjson.ParseBytes(transformRaw)
-	if parsedJSON.Get("output").Exists() {
```

Comment:
this block of code is not required anymore?

Reply:
Yea. Not required afaik. @SumanthPuram can validate

Reply:
@sivashanmukh Can you test this with a user-transformation?

There are 4 cases we need to validate.

User transformation - success/failure
Dest transformation - success/faulure

Reply:
A couple more:
Source transformation - success/failure

Reply:
Test with the latest transformer. You can the see the output that the Transformer is returning in versionedRouter.js.

Reply:
I have tested it throughly today, and this is not required.

`processor.go: func (proc *HandleT) getDestTransformerEvents(....
`
The output in the transformer response is handled here and only its contents are stored in the job which is picked by router. So this is not required.

Reply:
Perfect